src/Neo.CLI/CLI/MainService.cs

```diff
@@ -392,25 +391,32 @@ public async void Start(CommandLineOptions options)
                     }
                     else
                     {
-                        DisplayError("DLL not found, please get libleveldb.dll.");
+                        DisplayError("DLL not found, please get libleveldb.dll.",
+                            "Download from https://github.com/neo-ngd/leveldb/releases");
                     }
                 }
-                else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                else if (OperatingSystem.IsLinux())
+                {
+                    DisplayError("Shared library libleveldb.so not found, please get libleveldb.so.",
+                        "Use command \"sudo apt-get install libleveldb-dev\" in terminal or download from https://github.com/neo-ngd/leveldb/releases");
+                }
+                else if (OperatingSystem.IsMacOS() || OperatingSystem.IsMacCatalyst())
                 {
                     DisplayError("Shared library libleveldb.dylib not found, please get libleveldb.dylib.",
-                        "From https://github.com/neo-project/neo/releases");
+                        "Use command \"brew install leveldb\" in terminal and download from https://github.com/neo-ngd/leveldb/releases");
```

Comment:
Let's stay consistent with the above. use `or` instead of `and`

```suggestion
                        "Use command \"brew install leveldb\" in terminal or download from https://github.com/neo-ngd/leveldb/releases");
```

Reply:
You need to do both, for `Snappy compression library`.